### DISCUSSION TITLE
Release-5.0.0 delay

### DISCUSSION THREAD

**Discussion by pallakartheekreddy** [Sunbird-inQuiry/Community#26] [General Discussions]
Hello All,

We were supposed to provide the Build Block testing QA sign-off of R 5.0.0 by July. But due to the development of a new inquiry environment, unfortunately, we are delayed. Apologies for the delay. Two enhancements have been deployed and testing is in progress. Henceforth we will provide the Sign off max by Monday, 8th August 2022.

@surendrasinghs @chitranshu-keshav @ashokreddy1208 @Nadiya496

**Comment by Nadiya496** [comment]
Hi @pallakartheekreddy and @surendrasinghs ,

We have given the Building block QA testing Sign Off for the release 5.0.
cc: @chitranshu-keshav @Aparna-karri591 @anunagaraj
